agent/stats/engine.go

```diff
@@ -71,6 +71,7 @@ type Engine interface {
 	GetInstanceMetrics() (*ecstcs.MetricsMetadata, []*ecstcs.TaskMetric, error)
 	ContainerDockerStats(taskARN string, containerID string) (*types.StatsJSON, *NetworkStatsPerSec, error)
 	GetTaskHealthMetrics() (*ecstcs.HealthMetadata, []*ecstcs.TaskHealth, error)
+	GetServiceConnectStats() error
```

Comment:
We should probably return the stats here since tacs client can not access the stats stored in `taskToServiceConnectStats` map?

Reply:
You are right. I will implement that in a subsequent PR where I parse the stats and return the metrics following the format of the general metrics in the TACS model. I will add a TODO here.

Reply:
I made the comment assuming that we would call to `GetServiceConnectStats()` from [TACS client](https://github.com/aws/amazon-ecs-agent/blob/fe79b107f3b204ca1a831b1d15424d12d2948bdf/agent/tcs/client/client.go#L204-L243) but on second thought, [ServiceConnectMetrics](https://github.com/aws/amazon-ecs-agent/blob/feature/service-connect/agent/tcs/model/ecstcs/api.go#L994)  is returned as part of `GetInstanceMetrics()` call 
so I guess it is alright if `GetServiceConnectStats()` does not return stats since  `GetInstanceMetrics()`  has to access `taskToServiceConnectStats`.